.github/workflows/release.yml

```diff
@@ -27,30 +27,25 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
         run: |
-          gh --repo mitchellh/gon release download --pattern "gon_macos.dmg" --dir /tmp
-          sudo hdiutil attach /tmp/gon_macos.dmg
-          sudo mkdir -p $HOME/.local/bin
-          sudo cp /Volumes/gon/gon $HOME/.local/bin/gon
-          echo "$HOME/.local/bin" >> $GITHUB_PATH
+          gh --repo mitchellh/gon release download --pattern "gon_macos.zip" --dir /tmp
+          sudo unzip /tmp/gon_macos.zip -d /usr/local/bin
+          sudo chmod +x /usr/local/bin/gon
 
       - name: Import GPG key
         id: import_gpg
-        uses: crazy-max/ghaction-import-gpg@v4
+        uses: crazy-max/ghaction-import-gpg@v5
         with:
           gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
           passphrase: ${{ secrets.GPG_PASSPHRASE }}
-          fingerprint: ${{ secrets.GPG_FINGERPRINT }}
 
       - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v4
         with:
           distribution: goreleaser
           version: latest
           args: release --clean
-          workdir: ./cmd/ssh-sign/
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
```

Comment:
Same as above:

This needs to be added back in as a sub GPG key is being used.

Reply:
Likewise done.